doc/examples.xml

```diff
@@ -309,3 +309,44 @@ gap> GeneralisedPetersenGraph(IsMutableDigraph, 9, 4);
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="KnightsGraph">
+<ManSection>
+  <Oper Name="KnightsGraph" Arg="[filt, ]n, k"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> and <A>k</A> are both positive integers, then this operation returns
+    the <E>Knight's Graph</E> for a n by k board. <P/>
```

Comment:
Do we need to use capitalisation here?

Reply:
I agree on the consistency front. Note that here, and in the arguments, we use `n` and `k` for the parameters of the knight's graph, but in the Wikipedia quotation, we use `m` and `n`. I think it would be best for these to match. (Especially as the `m` and `n` in the Wikipedia text already have `<A>` tags around them, which indicates that those are the names of the arguments.